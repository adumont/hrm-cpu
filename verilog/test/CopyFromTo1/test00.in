53 fa
17 9c
37 be
59 45
47 bb
27 d6
26 7b
42 58
38 d5
03 6f
26 d8
09 c8
52 d5
26 96
47 f4
54 bd
04 27
56 2d
53 0a
17 37
47 7e
53 2b
01 cd
12 3b
09 dc
37 a4
37 34
46 a6
47 ef
56 76
51 f5
03 37
02 5e
27 e3
32 65
43 f5
32 c3
26 14
55 4c
15 ab
