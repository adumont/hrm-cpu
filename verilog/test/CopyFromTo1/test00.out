fa
9c
be
45
bb
d6
7b
58
d5
6f
d8
c8
d5
96
f4
bd
27
2d
0a
37
7e
2b
cd
3b
dc
a4
34
a6
ef
76
f5
37
5e
e3
65
f5
c3
14
4c
ab
